client/src/App.js

```diff
@@ -17,13 +19,13 @@ class App extends Component {
   }
 
   render() {
-    const { menuItems } = this.state;
+    const { menuItems } = this.state; // eslint-disable-line
     return (
       <Switch>
         <Route exact path="/" component={Loginpage} />
         <Route exact path="/recommend" component={Recommendpage} />
+        <Route exact path="/menuitem" component={MenuItempage} />
```

Comment:
in the future there'll probably be a separate route to each different menu item, so we'll have to do that later on